Dockerfile

```diff
@@ -0,0 +1,15 @@
+###################################################
+# This Dockerfile is used by the docker-compose.yml 
+# file to build the development container.
+# Do not make any changes here unless you know what
+# you are doing.
+###################################################
+
+FROM node:16-bullseye as dev 
+RUN apt-get update -y && apt-get upgrade -y && apt-get install -y openssl
+WORKDIR /app
+COPY .yarn ./.yarn
+COPY yarn.lock package.json .yarnrc.yml ./
+COPY . .
+RUN yarn
```

Comment:
Is this still necessary if the working directory is mounted in?

Reply:
Ah, probably not. I was testing some things and threw some redundancy in there. Let me test it without real quick and I'll get back to you.

Reply:
@j-f1 tested on my end and verified those two lines are not needed. Feel free to remove before merging.